src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,60 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
```

Comment:
```suggestion
public class Engine implements Cloneable {
```
I don't see clone method overrided here